Exiled.CustomRoles/API/Features/Extensions/ParserExtension.cs

```diff
@@ -0,0 +1,66 @@
+// -----------------------------------------------------------------------
+// <copyright file="ParserExtension.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.CustomRoles.API.Features.Extensions
+{
+    using System;
+
+    using Exiled.API.Features;
+    using Exiled.CustomRoles.API.Features.Parsers;
+
+    using YamlDotNet.Core;
+    using YamlDotNet.Core.Events;
+
+    /// <summary>
+    /// Extensions for <see cref="ParsingEventBuffer"/>.
+    /// </summary>
+    public static class ParserExtension
+    {
+        /// <summary>
+        /// Tries to find a valid mapping entry.
+        /// </summary>
+        /// <param name="parser">The <see cref="ParsingEventBuffer"/> parser.</param>
+        /// <param name="selector">The selector.</param>
+        /// <param name="key">The key found.</param>
+        /// <param name="value">The value found.</param>
+        /// <returns><see langwod="true"/> when a valid mapping entry is found.</returns>
```

Comment:
1. Fix typo in "langwod"
2. `<see langword="true"/> when a valid mapping entry is found; otherwise, <see langword="false">.` is better, since we should use .NET and C# docs style pattern.